cmd/govim/internal/fswatcher/os_darwin.go

```diff
@@ -68,6 +71,10 @@ func New(gomodpath string, tomb *tomb.Tomb) (*FSWatcher, error) {
 				event := events[i]
 				path := filepath.Join(mountPoint, event.Path)
 
+				if skipDir(path) {
```

Comment:
Is this guaranteed to be a directory path we are passing here?

Reply:
No, it could be directories or files. The function name came from the fact that it did check the directory part of any path provided, but that's just an implementation detail. Maybe just calling it `filter(...)` is more clear.
```go
if filter(path) {
```

Reply:
Agreed